src/Components/UpgradeRisksTable/AlertsList.js

```diff
@@ -0,0 +1,89 @@
+import { Flex, Icon } from '@patternfly/react-core';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+import InfoCircleIcon from '@patternfly/react-icons/dist/esm/icons/info-circle-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-triangle-icon';
+import {
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@patternfly/react-table';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useGetUpgradeRisksQuery } from '../../Services/SmartProxy';
+
+export const ALERTS_SEVERITY_ICONS = {
+  critical: (
+    <Icon status="danger">
+      <ExclamationCircleIcon />
+    </Icon>
+  ),
+  warning: (
+    <Icon status="warning">
+      <ExclamationTriangleIcon />
+    </Icon>
+  ),
+  info: (
+    <Icon status="info">
+      <InfoCircleIcon />
+    </Icon>
+  ),
+};
+
+export const ALERTS_SEVERITY_LABEL = {
+  critical: (
+    <Flex alignItems={{ default: 'alignItemsCenter' }}>
+      {ALERTS_SEVERITY_ICONS['critical']} <b>Critical</b>
+    </Flex>
+  ),
+  warning: (
+    <Flex alignItems={{ default: 'alignItemsCenter' }}>
+      {ALERTS_SEVERITY_ICONS['warning']} <b>Warning</b>
+    </Flex>
+  ),
+  info: (
+    <Flex alignItems={{ default: 'alignItemsCenter' }}>
+      {ALERTS_SEVERITY_ICONS['info']} <b>Info</b>
+    </Flex>
+  ),
+};
+
+export const ALERTS_SEVERITY_ORDER = ['critical', 'warning', 'info'];
+
+const AlertsList = () => {
+  const { clusterId } = useParams();
+  const { data } = useGetUpgradeRisksQuery({ id: clusterId });
+  const { alerts = [] } =
+    data?.upgrade_recommendation?.upgrade_risks_predictors || {};
+
+  return (
+    <TableComposable
```

Comment:
When comparing to the mocks I see a few links/icons that arent shown here.
Your version:
<img width="1231" alt="Screen Shot 2023-04-14 at 9 16 19 AM" src="https://user-images.githubusercontent.com/60629070/232054118-6e90c465-c146-44a8-8ad6-430047ef1511.png">

mock version:
<img width="713" alt="Screen Shot 2023-04-14 at 9 17 15 AM" src="https://user-images.githubusercontent.com/60629070/232054356-01955bfa-ca00-40d3-bd55-5f33a910a3ab.png">

Reply:
good catch. there is no final agreement on the docs URL, so I'd wait for folks first to decide and then will add it via separate PR most probably